lib/websocket.js

```diff
@@ -392,8 +392,13 @@ class WebSocket extends EventEmitter {
   }
 }
 
+WebSocket.CONNECTING = 0
+WebSocket.OPEN = 1
+WebSocket.CLOSING = 2
+WebSocket.CLOSED = 3
+
```

Comment:
Why this change? It is already done by ` Object.defineProperty(WebSocket, readyState, descriptor);` below.

Reply:
VS inteligens could not figure out what dynamically assigned properties meant

Reply:
I prefer to not have changes only required for TypeScript / Visual Studio Code IntelliSense.

Reply:
Do you have any other jsDoc solution up your sleeve that describes WebSocket to have this properties?

Reply:
It's weird but something like this should work:

```
/**
 * Class representing a WebSocket.
 *
 * @property {Number} CONNECTING  description
 */
```

Reply:
forgot about that thing